pkgs/applications/networking/flottbot/default.nix

```
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub, nixosTests, nix-update-script }:
+buildGoModule rec {
+  pname = "flottbot";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "target";
+    repo = pname;
+    rev = "${pname}-${version}";
```

Currently the url syntax is wrong, you only have to add the version in this case ande the wrapper does the rest.
You usally give the hash of a commit or the version number of a tag, you can look at other packages using the github wrapper with: ```rg fetchFromGitHub``` to get some examples
```suggestion
    rev = version;
```